packages/expo-video/android/src/main/java/expo/modules/video/records/VideoThumbnailOptions.kt

```diff
@@ -0,0 +1,10 @@
+package expo.modules.video.records
+
+import expo.modules.kotlin.records.Field
+import expo.modules.kotlin.records.Record
+import java.io.Serializable
+
+class VideoThumbnailOptions(
+  @Field val maxWidth: Int? = null,
+  @Field val maxHeight: Int? = null
+) : Record, Serializable
```

Comment:
```suggestion
) : Record
```
Is there a reason why you ended up making it serializable?